hooks/post/30-upgrade.rb

```diff
@@ -80,6 +87,7 @@ def fail_and_exit(message)
     upgrade_step :import_puppet_modules, :long_running => true
     upgrade_step :import_subscriptions, :long_running => true
     upgrade_step :remove_elasticsearch
+    upgrade_set :remove_docker_v1_content, :long_running => true
```

Comment:
@ehelms heh urite.

@bbuckingham i spoke with @ehelms and @jlsherrill over lunch and they seemed to think it would be fine before or after. If you look at the rake task, it's identifying the repositories through Runcible, rather than through the Katello model, then just telling the repository to destroy itself along with its clones.